audit/package.json

```diff
@@ -22,6 +22,7 @@
     "@libero/event-bus": "../lib/event-bus",
     "@libero/libero-events": "../lib/libero-events",
     "@libero/migrator": "../lib/migrator",
+    "auth-utils": "../lib/auth-utils",
```

Comment:
For some reason this is restored by `yarn upgrade:internal` so leaving in

Reply:
most likely a problem with the yarn.lock not updating correctly. We probably should look into fixing it because `auth-utils` is an existing npm package